### DISCUSSION TITLE
Attribute “xmlns:b” not allowed here.

### DISCUSSION THREAD

**Discussion by Saadimabroukii** [community/community#31088] [Other Feature Feedback, Questions, & Ideas]
Attribute “xmlns:b” not allowed here.

**Comment by Saadimabroukii** [comment]
Line: 2 Column: 16 - 215
"...TYPE html> <html class='v2' dir='rtl' lang='ar' xmlns='http://www.w3.org/1999/xhtml' xmlns:b='http://www.google.com/2005/gml/b' xmlns:data='http://www.google.com/2005/gml/data' xmlns:expr='http://www.google.com/2005/gml/expr'> <head..."

**Comment by airtower-luna** [comment]
Sounds like your XML or HTML structure is broken. How to fix that is impossible to say without seeing the actual code.
